content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md

```diff
@@ -123,4 +123,4 @@ kubectl delete secret db-user-pass
 
 - Read more about the [Secret concept](/docs/concepts/configuration/secret/)
 - Learn how to [manage Secrets using kubectl](/docs/tasks/configmap-secret/managing-secret-using-kubectl/)
-- Learn how to [manage Secrets using config file](/docs/tasks/configmap-secret/managing-secret-using-config-file/)
\ No newline at end of file
+- Learn how to [manage Secrets using config file](/docs/tasks/configmap-secret/managing-secret-using-config-file/)
```

Comment:
nit: What has changed on this line?

Reply:
I think this was just the Github editor automatically adding a newline at the end of the file.
Do you need me to revert this?

Reply:
It's good practice not to change unneeded sections like this, but if it's too much of a pain to revert it should be fine as is.